src/openforms/js/components/admin/form_design/AuthPluginField.js

```diff
@@ -24,15 +24,16 @@ const AuthPluginField = ({availableAuthPlugins, selectedAuthPlugins, onChange, e
     );
 
     return (
-      <Checkbox
-        key={plugin.id}
-        name={plugin.label}
-        value={plugin.id}
-        label={label}
-        onChange={onChange}
-        checked={selectedAuthPlugins.includes(plugin.id)}
-        noVCheckbox
-      />
+      <li key={plugin.id}>
+        <Checkbox
+          name={plugin.label}
+          value={plugin.id}
+          label={label}
```

Comment:
can we inline that variable from above? the nested fragments are making my head fuzzy